spec/util/bundler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+RSpec.describe PDK::Util::Bundler do
+  before(:each) do
+    allow(PDK::CLI::Exec).to receive(:execute).and_return({exit_code: 0})
+  end
+
+  describe '.ensure_bundle!' do
+    context 'when there is no Gemfile' do
+      before(:each) do
+        allow(File).to receive(:file?).with(/Gemfile$/).and_return(false)
+      end
+
+      it 'does nothing' do
```

Comment:
I guess we just need to figure out how the most graceful way to handle the lack of a Gemfile is. We can change the method name to something like `require_bundle!` to make it clear that the tool should exit 1 if there is no Gemfile present in the module.